.eslintrc.js

```diff
@@ -53,6 +53,11 @@ module.exports = {
     parserOptions: {
         ecmaVersion: 2020
     },
+    settings: {
+        jsdoc: {
+            mode: "typescript"
```

Comment:
Why does this need to be changed from the default of `jsdoc`?

Reply:
it reports: "Invalid JSDoc tag name "template" jsdoc/check-tag-names"
as we are using tag `template`, see https://github.com/gajus/eslint-plugin-jsdoc#check-tag-names

Reply:
added a comment. 😄